### PR TITLE
Centralize the handling of browser.OpenURL

### PR DESCRIPTION
**Pull request by afbjorklund**
So we can catch when xdg-open not installed

Might help with #7168

### CODE REVIEW

**Comment by afbjorklund** [comment]
Right, have to come up with a nice way of handling this.
Maybe just a text saying "please open a browser at ..."

**Comment by afbjorklund** [comment]
Went with emoji instead

File: pkg/minikube/browser/browser.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	"k8s.io/minikube/pkg/minikube/out"
+)
+
+// OpenURL opens a new browser window pointing to URL.
+func OpenURL(url string) error {
+	if runtime.GOOS == "linux" {
+		_, err := exec.LookPath("xdg-open")
+		if err != nil {
+			out.T(out.URL, url)
+			return nil
```
Comment (medyagh) [inline pkg/minikube/browser/browser.go]: how about some additional wording to the URL ?
how does this look like in the output ?

**Comment by medyagh** [comment]
> Went with emoji instead

do u mind pasting the ouput in the PR description so we see how it looks like?

**Comment by afbjorklund** [comment]
I don't have the environment set up to reproduce it, but I _guess_ it would look something like:

```
🎉  Opening service default/hello-minikube in default browser...
👉  http://10.128.0.21:31284
```

**Pull request merged**